ci/build_container/build_container_common.sh

```diff
@@ -14,4 +14,12 @@ if [[ "$(uname -m)" == "x86_64" ]]; then
   curl --location --output /usr/local/bin/bazel https://github.com/bazelbuild/bazelisk/releases/download/v${VERSION}/bazelisk-linux-amd64 \
     && echo "$SHA256  /usr/local/bin/bazel" | sha256sum --check \
     && chmod +x /usr/local/bin/bazel
+
+  # ninja
+  VERSION=1.8.2
```

Comment:
1.8.2 is from September 2017. Any reason your not using the latest Ninja release 1.9.0? https://github.com/ninja-build/ninja/releases/tag/v1.9.0

Reply:
As far as I remember, v1.9.0 was built using newer glibc, and it won't load on CentOS.